### ISSUE TITLE
`Error: one snapshot has incorrect values (--inline-snapshot=fix)` is confusing when all tests pass

### ISSUE THREAD

**Issue by samuelcolvin** [15r10nk/inline-snapshot#116]
Again, thank you for inline-snaphots, we use it all the time and it's great.

I think use of the word "Error" is pretty confusing here:

<img width="772" alt="image" src="https://github.com/user-attachments/assets/73408bbb-aa3f-4eec-9508-6e47b72751cd">

If I understand this correctly, inline-snapshots is really just suggesting it would format one of the tests differently? If so it should say that, not "Error".

... or am I miss understanding?

**Comment by samuelcolvin** [comment]
For context, this is the change inline-snapshots suggests if I run `--inline-snapshot=review`:

```
code/pydantic-data-platform 0 ➤ uv run pytest src/services/fusionfire/fusionfire_tests/test_ingest.py --inline-snapshot=review
===================================================================== test session starts ======================================================================
platform darwin -- Python 3.12.6, pytest-8.3.2, pluggy-1.5.0
rootdir: /Users/samuel/code/pydantic-data-platform
configfile: pyproject.toml
plugins: recording-0.13.2, inline-snapshot-0.12.1, time-machine-2.15.0, playwright-0.5.1, logfire-0.53.0, devtools-0.12.2, anyio-4.4.0, base-url-2.1.0, timeout-2.3.1, pretty-1.2.0, xdist-3.6.1, requests-mock-1.12.1
collected 4 items                                                                                                                                              

src/services/fusionfire/fusionfire_tests/test_ingest.py ....                                                                                             [100%]
======================================================================= inline snapshot ========================================================================
──────────────────────────────────────────────────────────────────────── Fix snapshots ─────────────────────────────────────────────────────────────────────────
╭────────────────────────────────────────────────── src/services/fusionfire/fusionfire_tests/test_ingest.py ───────────────────────────────────────────────────╮
│ @@ -143,7 +143,7 @@                                                                                                                                          │
│                                                                                                                                                              │
│                  'start_timestamp:timestamp[us, tz=UTC]': datetime.datetime(                                                                                 │
│                      1970, 1, 1, 0, 0, 1, tzinfo=zoneinfo.ZoneInfo(key='UTC')                                                                                │
│                  ),                                                                                                                                          │
│ -                'created_at:timestamp[us, tz=UTC]': IsNow(tz=zoneinfo.ZoneInfo(key='UTC')),                                                                 │
│ +                'created_at:timestamp[us, tz=UTC]': datetime.datetime(2024, 9, 21, 15, 14, 47, 738080, tzinfo=zoneinfo.ZoneInfo(key="UTC")),                │
│                  'trace_id:large_string': '00000000000000000000000000000001',                                                                                │
│                  'span_id:large_string': '0000000000000001',                                                                                                 │
│                  'parent_span_id:large_string': None,                                                                                                        │
╰──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯
do you want to fix these snapshots? [y/n] (n): n

Results (17.33s):
         4 passed
code/pydantic-data-platform 0 ➤ 
```

Also, there seems to be a typo, looks like the sentence

```
You can also use --inline-snapshot=review to approve the changes interactiv
```

is unfinished.

**Comment by 15r10nk** [comment]
This took me a while to reproduce, because the branch which I'm currently working on fixed this issue :smile: .

The reason is probably the time difference between the call of now() and the comparison. 

Can you try this branch to verify that the problem is fixed?
https://github.com/15r10nk/inline-snapshot/pull/114

**Comment by 15r10nk** [comment]
fixed in 0.13.2
